implementations.md

```diff
@@ -9,15 +9,15 @@ title: VC Implementations
   - [Verifiable Credentials JS Library](https://github.com/digitalbazaar/vc-js) (DI)
   - [Verifiable Data Credential JS Library with JWT](https://github.com/transmute-industries/verifiable-data/tree/main/packages/vc.js) (DI & JWT)
   - [W3C Verifiable Credentials and Presentations in JWT format](https://github.com/decentralized-identity/did-jwt-vc) (JWT)
-  - [Cryptographic Primitives used for DID](https://github.com/microsoft/VerifiableCredentials-Crypto-SDK-Typescript) (JWT)
+  - [Cryptographic Primitives used for DID](https://github.com/microsoft/VerifiableCredential-SDK-Android, https://github.com/microsoft/VerifiableCredential-SDK-iOS) (JWT)
```

Comment:
@iherman -- This is invalid Markdown. The human-facing title `Cryptographic Primitives used for DID` cannot have two URLs in the `href` that results from the Markdown-to-HTML transformation. Please look to [my later suggestion and subsequent conversation with @Sakurann](https://github.com/w3c/verifiable-credentials/pull/41#discussion_r889062603).

Reply:
This came from an earlier change, it has been changed right now.